develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -31,22 +52,41 @@ Allows you to define a newly created dialog box item as a button.
 Example:
 
 ```csharp
-UIBlockDefinition blockItem = new UIBlockDefinition();
-blockItem.Type = UIBlockType.Button;
-...
+UIBlockDefinition blockItem = new UIBlockDefinition
+{
+  Type = UIBlockType.Button,
+  Text = "Enter",
+  DestVar = "ButtonVar"
+};
 MyDialogBox.AppendBlock(blockItem);
+
+MyDialogBox.RequireResponse = true;
+
+var results = engine.ShowUI(MyDialogBox);
+
+if (results.WasButtonPressed("ButtonVar"))
+{
+  //Do something
+}
 ```
 
 ## Calendar
 
 Allows you to define a newly created dialog box item as a calendar control.
 
+To have a certain initial value in the calendar, use the [InitialValue](xref:Skyline.DataMiner.Automation.UIBlockDefinition#Skyline_DataMiner_Automation_UIBlockDefinition_InitialValue) property. In the *Calendar* UIBlockType, the value is expected to be a string in the "(dd/MM/yyyy HH:mm:ss)" format.
+
 Example:
 
 ```csharp
-UIBlockDefinition blockItem = new UIBlockDefinition();
-blockItem.Type = UIBlockType.Calendar;
-...
+UIBlockDefinition blockItem = new UIBlockDefinition
+{
+  Type = UIBlockType.Calendar,
+  InitialValue = DateTime.Now.ToString("dd/MM/yyyy HH:mm:ss"),
+  DestVar = "InputCalendar",
+  Row = 1,
```

Comment:
We could remove these row & column lines in all examples to safe some lines. These are generic fields that (should) behave the same for all blocks. The same could be said for the DestVar tbh, maybe only fill it in when the  block definition is accompanied by checking logic?